src/stylesheets/components/_alerts.scss

```diff
@@ -41,6 +41,7 @@ $icon-size: units(4);
     margin-top: units(2);
   }
 
+  // TODO: why is this not simply a border?
```

Comment:
it's so you can have a plain alert with no border, try it with `usa-alert`.

Reply:
Even `usa-alert` has a gray border. If there's a borderless alert, I can't track it down.  And even if there was one, the benefit of this technique (better text alignment between bordered and non-bordered alerts?) seems pretty minimal for the extra css required for an absolutely-positioned bar. 🤷‍♀️

Reply:
Not saying this is a first-priority change by any means — just something I noticed